pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -137,6 +138,7 @@ let
         "--sysconfdir=/etc"
         "--localstatedir=/var"
         "--enable-systemd"
+        "--enable-pam"
```

Comment:
What is the closure size increase by adding PAM to the build?

Reply:
The effect of `--enable-pam` is adding `lib/security/pam_zfs_key.so` which is about 73 KB.

Reply:
Awesome